api/tests/unit/integrations/test_unit_integration.py

```diff
@@ -0,0 +1,132 @@
+from unittest import mock
+from unittest.case import TestCase
+
+import pytest
+
+from environments.identities.models import Identity
+from environments.models import Environment
+from integrations.amplitude.models import AmplitudeConfiguration
+from integrations.integration import identify_integrations
+from integrations.segment.models import SegmentConfiguration
+from organisations.models import Organisation, OrganisationRole
+from projects.models import Project
+from util.tests import Helper
+
+
+@pytest.mark.django_db
+class HelperTestCase(TestCase):
+    identifier = "user1"
+
+    def setUp(self):
+        user = Helper.create_ffadminuser()
```

Comment:
Possibly this is just old code that has been moved but let's not use this horrible old `Helper` class please. I'm pretty sure there is a fixture in `pytest-django` for a superuser?